File: fib.ml
let rec fib i =
  if i <= 1 then 1 else fib (i - 1) + fib (i - 2);;
let rec fib n = match n with
| 0 -> 0
| 1 -> 1
| i -> fib (i - 1) + fib (i - 2);;
print_int (fib 38);;
print_newline ();;

File: fib.c
#include <stdio.h>

static int fib(int n) {
  if (n <= 1) return 1;
  return fib(n - 1) + fib(n - 2);
}

int main(void) {
  printf("%d \n", fib(38));
  return 0;
}
